src/OptiHPLCHandler/empower_instrument_method.py

```diff
@@ -131,6 +143,7 @@ class ColumnHandlerMethod(InstrumentMethod):
 
     @property
     def column_temperature(self):
+        """The column temperature."""
```

Comment:
in sample manager

Reply:
No, this is in the parent method, so it is in both the sample manager and the column manager.